### ISSUE TITLE
RegExp sub-questions in Multianswer/Cloze type questions with Moodle 3.3

### ISSUE THREAD

**Issue by ffeytout** [ndunand/moodle-qtype_regexp#3]
Hi,
Your **renderer.php** and **questiontype.php** files  are still for Moodle 2.9 and do not work with 3.3. 

Can you give us a 3.3 compatible version please ? 
Thank you.

**Comment by ndunand** [comment]
Hi,

Thanks for reporting this. Can you please be more specific. Our tests showed this working in Moodle 3.3. What does not work, exactly?

**Comment by ffeytout** [comment]
Hi and sorry for the delay. 
Please excuse my former post, I was in a hurry. I got your plugin to work with Multianswer/Cloze type by utilising the 2.7 version of the modified files.
The problem started with this doc page **[https://docs.moodle.org/31/en/Regular_Expression_Short-Answer_question_type#Inserting_RegExp_sub-questions_in_Cloze_type_questions](https://docs.moodle.org/31/en/Regular_Expression_Short-Answer_question_type#Inserting_RegExp_sub-questions_in_Cloze_type_questions)** . 
The links are broken and if you save target as you will have a wonderful php file containing only "404: Not Found". 

So in conclusion, your plugin and modified files are good but the doc page is outdated and don't serve the proper files anymore.
Cheers,

**Comment by ndunand** [comment]
Hi,
Thanks for looking into this.
